Model/Util/Service/DefaultsService.swift

```diff
@@ -0,0 +1,21 @@
+//
+//  Defaults.swift
+//  Defaults
+//
+//  Created by Luca D'Alberti on 9/27/16.
+//  Copyright © 2016 dalu93. All rights reserved.
+//
+import Foundation
+
+public struct DefaultKey<T>: Equatable {
+    public var name: String
+    public var defaultValue: T
+    public init(name: String, defaultValue: T) {
+        self.name = name
+        self.defaultValue = defaultValue
+    }
+}
+
+public func ==<T>(lhs: DefaultKey<T>, rhs: DefaultKey<T>) -> Bool {
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/eifandevs/Amby/blob/13b22eabc34a3da4be6d8d051199f3a006a4b490/Model/Util/Service/DefaultsService.swift#L19"></span>Operators should be surrounded by a single whitespace when defining them.<br />
<code>operator_whitespace</code> <code>DefaultsService.swift:19</code></td>
    </tr>
  </tbody>
</table>